### PR TITLE
Upgrade backtintime to 1.2.1 and add qt5 gui package

### PR DESCRIPTION
**Pull request by alpicoid**
please merge these 2 packages.
Regards

### CODE REVIEW

**Comment by Chocimier** [comment]
Does it work for you? I got `dbus.exceptions.DBusException: org.freedesktop.DBus.Error.NameHasNoOwner: Could not get owner of name 'net.launchpad.backintime.serviceHelper': no such name` when running.

**Comment by alpicoid** [comment]
@Chocimier, sorry there was an error in the commit (wrong file) and the qt Gui template was wrong. It's fixed now and it works for me now.

File: srcpkgs/backintime-qt/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+archs=noarch
+wrksrc=backintime-${version}
+build_wrksrc="qt"
+build_style=configure
+configure_args="--python3"
+pycompile_dirs="/usr/share/backintime"
+hostmakedepends="python3"
+depends="python3 rsync python3-dbus python3-PyQt5 python3-PyQt5-dbus libnotify xdpyinfo backintime"
+short_desc="Qt Gui for backintime"
+maintainer="Juan RP <xtraeme@voidlinux.eu>"
```
Comment (Chocimier) [inline srcpkgs/backintime-qt/template]: When adding new packages you should become maintainer.

File: srcpkgs/backintime-qt/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+version=1.2.0
+revision=1
+archs=noarch
+wrksrc=backintime-${version}
+build_wrksrc="qt"
+build_style=configure
+configure_args="--python3"
+pycompile_dirs="/usr/share/backintime"
+hostmakedepends="python3"
+depends="python3 rsync python3-dbus python3-PyQt5 python3-PyQt5-dbus libnotify xdpyinfo backintime"
```
Comment (Chocimier) [inline srcpkgs/backintime-qt/template]: As it uses backintime, its dependencies are not needed here.

File: srcpkgs/backintime-qt/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="Qt Gui for backintime"
+maintainer="Juan RP <xtraeme@voidlinux.eu>"
+license="GPL-2"
```
Comment (Chocimier) [inline srcpkgs/backintime-qt/template]: Find out whether it is GPL-2.0 only or GPL-2.0-or-later, and update. Same for backintime.

File: srcpkgs/backintime/template [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 configure_args="--python3"
-hostmakedepends="python3"
-depends="python3 rsync python3-dbus"
 pycompile_dirs="/usr/share/backintime"
-short_desc="A simple backup tool for Linux"
-maintainer="Juan RP <xtraeme@voidlinux.org>"
+hostmakedepends="python3"
+depends="python3 rsync python3-dbus python3-keyring"
+short_desc="Simple backup tool for Linux"
+maintainer="Juan RP <xtraeme@voidlinux.eu>"
```
Comment (Chocimier) [inline srcpkgs/backintime/template]: Juan recently resigned to maintain packages. Do you want to take it over?

Also `voidlinux.eu` is deprecated in favor of `.org`.

**Comment by alpicoid** [comment]
Thanks a lot for you review and advices. I applied the changes and squashed the commits.
The license is GPL 2.0 for the 2 packages. Ok to become the maintainer.

**Comment by Chocimier** [comment]
> The license is GPL 2.0

Set license to GPL-2.0-only then. Each package should have it's own commit. There are also conflicts in branches, please rebase onto current master. The rest is fine.

**Comment by alpicoid** [comment]
License set to GPL-2.0-only. 1 package per commit and rebase onto current master done.

**Comment by Chocimier** [comment]
There should also be (literally) `pycompile_version="$py3_ver"` in both templates to byte-compile for right python. While changing that, add version into commit title as per CONTRIBUTING.md.

**Comment by alpicoid** [comment]
I have updated to the latest version 1.2.1 and apply changes according to your guidance and CONTRIBUTING.md for the commit message. I hope it's fine now. Thanks for your help.

**Pull request merged**